ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/SurrogateTemplateIdCache.scala

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.http.dbbackend
+
+import com.daml.caching.SizedCache
+import com.daml.http.dbbackend.Queries.SurrogateTpId
+import com.daml.http.util.Logging.InstanceUUID
+import com.daml.logging.{ContextualizedLogger, LoggingContextOf}
+import com.daml.metrics.Metrics
+
+object SurrogateTemplateIdCache {
+  private val logger = ContextualizedLogger.get(getClass)
+  final val MAX_ENTRIES = 10L
```

Comment:
```suggestion
  final val MaxEntries = 10L
```
We usually use this format for constants.